benchmark.R

```diff
@@ -0,0 +1,182 @@
+#' ---
+#' title: "Testing package's functions"
+#' output: github_document
+#' ---
+
+#' ## Install and load the sPlot package
+devtools::install_github("andrewsiefert/sPlotOpenR", force = TRUE)
+library(sPlotOpenR)
+
+#' ### Load test data
+data("greece")
+
+#' ## Load and prepare additional data (non sPlotOpen) needed for testing 
+library(tidyverse)
+library(rnaturalearth)
+library(sf)
+sf_use_s2(use_s2 = FALSE)
+
+countries <- ne_countries(returnclass = "sf")
+europe <- countries %>% 
+  filter(continent == "Europe") %>% 
+  st_union()
+
+#' ## Check sPlotOpen documentation
+
+#' Check for package information
+?sPlotOpenR
+
+#' There is no general description of the package.
+
+??sPlotOpenR
+#' But there is good documentation for the functions.
+
+
+#' ## Testing the functions
+#' ### get_sPlot()
+#'
+#' With default options
+get_sPlot()
+
+#' It worked nicely, including the progress messages and the creation of the directories.
+#' However, all files in the iDiv repository were saved within the directory, in addition to the three table data (DT, header and CWM_CVM)
+#' For example, the pdf file Demo.pdf and the folder Schema. 
+#' I believe this is occurring because now we can only download all files from the iDiv portal.
+#' Also, I think this will be changed when the  argument metadata is implemented, right?
```

Comment:
That's right, when we have the ability to download individual files, I plan to add options to download individual tables and metadata or the full zipped folder.